plugins/Kaleidoscope-FocusSerial/src/kaleidoscope/plugin/FocusSerial.h

```diff
@@ -37,8 +46,22 @@ class FocusSerial : public kaleidoscope::Plugin {
   static constexpr char SEPARATOR = ' ';
   static constexpr char NEWLINE   = '\n';
 
-  bool handleHelp(const char *command,
-                  const char *help_message);
+#ifndef NDEPRECATED
+  DEPRECATED(FOCUS_HANDLEHELP)
+  bool handleHelp(const char *input, const char *help_message);
+#endif
+
+  bool inputMatchesHelp(const char *input);
+  bool inputMatchesCommand(const char *input, const char *expected);
+
+  EventHandlerResult printHelp() {
+    return EventHandlerResult::OK;
+  }
+  template<typename... Vars>
+  EventHandlerResult printHelp(const char *h1, Vars... vars) {
+    Runtime.serialPort().println((const __FlashStringHelper *)h1);
```

Comment:
Do you maybe want to add a `delayAfterPrint()` here?

Reply:
I do. Fixed, thanks!